tests/pubgrade/test_server.py

```diff
@@ -47,200 +52,221 @@
+        db=MongoConfig(**MONGO_CONFIG), endpoints=ENDPOINT_CONFIG
+    )
+    app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+        "repositories"
+    ].client = mongomock.MongoClient().db.collection
     for repository in MOCK_REPOSITORIES:
-        app.config['FOCA'].db.dbs['pubgradeStore']. \
-            collections['repositories'].client.insert_one(
-            repository).inserted_id
-    app.config['FOCA'].db.dbs['pubgradeStore']. \
-        collections['builds'].client = mongomock.MongoClient(
-    ).db.collection
-    app.config['FOCA'].db.dbs['pubgradeStore']. \
-        collections['builds'].client.insert_one(
-        MOCK_BUILD_INFO).inserted_id
-    app.config['FOCA'].db.dbs['pubgradeStore']. \
-        collections['builds'].client.insert_one(
-        MOCK_BUILD_INFO_2).inserted_id
+        app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+            "repositories"
+        ].client.insert_one(repository)
+    app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+        "builds"
+    ].client = mongomock.MongoClient().db.collection
+    app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+        "builds"
+    ].client.insert_one(MOCK_BUILD_INFO)
+    app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+        "builds"
+    ].client.insert_one(MOCK_BUILD_INFO_2)
     with app.test_request_context():
-        res = getBuilds.__wrapped__(MOCK_REPOSITORIES[1]['id'])
+        res = getBuilds.__wrapped__(MOCK_REPOSITORIES[1]["id"])
         assert isinstance(res, list)
         assert isinstance(res[0], dict)
 
 
 def test_getBuildInfo():
     app = Flask(__name__)
-    app.config['FOCA'] = \
-        Config(db=MongoConfig(**MONGO_CONFIG), endpoints=ENDPOINT_CONFIG)
-    app.config['FOCA'].db.dbs['pubgradeStore']. \
-        collections['repositories'].client = mongomock.MongoClient(
-    ).db.collection
+    app.config["FOCA"] = Config(
+        db=MongoConfig(**MONGO_CONFIG), endpoints=ENDPOINT_CONFIG
+    )
+    app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+        "repositories"
+    ].client = mongomock.MongoClient().db.collection
     for repository in MOCK_REPOSITORIES:
-        app.config['FOCA'].db.dbs['pubgradeStore']. \
-            collections['repositories'].client.insert_one(
-            repository).inserted_id
-    app.config['FOCA'].db.dbs['pubgradeStore']. \
-        collections['builds'].client = mongomock.MongoClient(
-    ).db.collection
-    app.config['FOCA'].db.dbs['pubgradeStore']. \
-        collections['builds'].client.insert_one(
-        MOCK_BUILD_INFO).inserted_id
+        app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+            "repositories"
+        ].client.insert_one(repository)
+    app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+        "builds"
+    ].client = mongomock.MongoClient().db.collection
+    app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+        "builds"
+    ].client.insert_one(MOCK_BUILD_INFO)
```

Comment:
Again, looking through this diff, I almost did not see that you dropped `inserted_id`. Probably there is other I missed.